pkg/find/vulnerabilities/trivy/converter.go

```diff
@@ -37,6 +38,14 @@ func (c *converter) Convert(reader io.Reader) (report starboard.VulnerabilityRep
 		return
 	}
 	report = c.convert(scanReports)
+
+	ref, err := name.ParseReference(imageRef)
```

Comment:
If you don't want to add more dependency, let me know, please. `moby/moby` also provides such a function, but it is heavier than `google/go-containerregistry`.